reference/procedural-proto-nodes.md

```diff
@@ -50,6 +50,7 @@ include external Lua modules.
 - Lua standard output and error streams are redirected on the Webots console
 (written respectively in regular and in red colors). This allows developers to
 use the Lua regular functions to write on these streams.
+- The lua-gd module is contained in Webots and can simply be imported using `local gd = require("gd")`. This module is very useful to manipulate images, it can for example be used to generate textures.
```

Comment:
I did put `lua-gd` as link to the documentation and can't therefore use the brackets, but I think it is ok as it is already not the same color than the rest of the text (because of the link), do you agree?